### DISCUSSION TITLE
Statsd Logs not showing up with docker compose

### DISCUSSION THREAD

**Discussion by Sangarshanan** [apache/superset#27121] [Q&A / Help]
#### How to reproduce the bug

Add the following lines to superset_config.py

```python
from superset.stats_logger import StatsdStatsLogger
STATS_LOGGER = StatsdStatsLogger(host='localhost', port=8125, prefix='superset')

```
and the extra param to gunicorn cmd in `run-server.sh`

`--statsd-host localhost:8125
`

I am also running a development server `FLASK_ENV` and `SUPERSET_ENV` set to `development`

BUT when i  build and run superset from scratch `Superset 0.0.0-dev` then the statd logger seems to show up locally, it however fails to show up in my docker compose

### Expected results

stats_logger should start logging major events like query timings

### Actual results

statsd logger does not show up.

### Environment

- browser type and version: Chrome Version 108.0.5359.94
- superset version: 1.51.2
- python version: Python 3.9.15
- node.js version: v18.11.0
- any feature flags active: FLASK_ENV, SUPERSET_ENV = development 

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stack traces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

stats_logger shows up when running locally but does not with docker-compose

**Comment by rusackas** [comment]
Maybe @nytai or @john-bodley can help with this one?

**Comment by rusackas** [comment]
This seems like it might warrant being moved to Q&A, if it is a configuration concern and not a bug, per se.
